google_shopping.js

```diff
@@ -2,4 +2,103 @@ var data = require("./products.json")
 
 // Write your solutions below
 
-console.log(data["items"]);
\ No newline at end of file
+//console.log(data["items"][0]['kind']);
+
+//QUESTION 1--------------------------
+
+
+var shoppingProducts = [];
+
+for (var i = 0; i < (data['items'].length); i++) {
+	if (data['items'][i]['kind'] === "shopping#product") {
+		shoppingProducts.push("Item " + i + " is a shopping#product.");
+	}
+}
+
+console.log(shoppingProducts);
+console.log(shoppingProducts.length + " items total."); 
+
+
+//Answer = 25, same as currentItemCount
+
+//QUESTION 2----------------------------
+
+
+var backOrdered = [];
+
+for (var i = 0; i < (data['items'].length); i++) {
+	for (var j = 0; j < data['items'][i]['product']['inventories'].length; j++) {
+		if ((data['items'][i]['product']['inventories'][j]['availability']) === "backorder") {
+			backOrdered.push("Item " + i + " has a backorder.");
+		}
+	}
+}
+
+console.log(backOrdered);
+console.log(backOrdered.length + " items total.");
+
+
+//Answer = 2 [Sony Alpha DSLR SLT-A77, Fujifilm FinePix FEAL 3D W3]
+
+//QUESTION 3----------------------------
+
+
+var manyImages = [];
+
+for (var i = 0; i < data['items'].length; i++) {
+	if (data['items'][i]['product']['images'].length > 1) {
+		manyImages.push("Item " + i + " has more than one image link.");
+	} 
+} 
+
+console.log(manyImages);
+console.log(manyImages.length + " items total.");
+
+
+//Answer = 6
+
+//QUESTION 4------------------------------
+
+
+var canonProducts = [];
+
+for (var i = 0; i < data['items'].length; i++) {
+	if (data['items'][i]['product']['brand'] === ("Canon" || "canon")) {
```

Comment:
Nice idea using an || operator here. Another way to ignore case is by using the .toLowerCase() method, like so:

```if (data['items'][i]['product']['brand'].toLowerCase() === "canon")```